config/routes.rb

```diff
@@ -244,6 +244,9 @@
     get 'terms' => 'terms#index'
   end
 
+  # Relay
+  get ".well-known/social-relay" => "social_relay#well_known"
```

Comment:
The `x-` prefix is a best practice introduced by a lot of other protocols (mail, http, ...). But if you're going to get it into the registry (hint, wellknown-uri-review@ietf.org), good luck with that! :)